source/wp-content/themes/wporg-parent-2021/sass/block-styles.scss

```diff
@@ -121,3 +121,9 @@
 		}
 	}
 }
+
+.is-style-serif {
+	font-family: var(--wp--custom--heading--typography--font-family);
+	font-size: var(--wp--preset--font-size--heading-5);
+	line-height: var(--wp--custom--heading--level-5--typography--line-height);
```

Comment:
I realise these match our use case, but it feels strange to be setting them too; I'd expect to only be changing the `font-family`. Is there a reason not to use the standard typography text size controls for this?

Reply:
This just sets up the default, because I think it's safe to assume that when someone wants the serif size, they also want the slightly larger font. The typography controls will still work to set other sizes if this assumption is wrong.